scripts/squad-run-node.sh

```diff
@@ -243,31 +243,6 @@ cleanup() {
     unset TENCENTCLOUD_SECRET_KEY
 }
 
-# TODO: delete this function when new version of tencentcloud provider is released
```

Comment:
you removed the firewal cli commands but I don't see the addition of the correct ones, where is the rule removal done?

Reply:
now it uses the firewall_template to set the firewall rules.
you can refer to the codes tencentcloud_lighthouse_firewall_template in main.tf line 48

Reply:
understood, this is much better.